CMake/setup_KWIVER.sh.in

```diff
@@ -1,20 +1,19 @@
+#!/bin/bash
 #
 # Script to source to setup the KWIVER environment
 #
 this_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 export VG_PLUGIN_PATH=$this_dir
 export PATH=$this_dir/bin:$PATH
-export @LIBRARY_PATH_VAR@=$this_dir/@KWIVER_DEFAULT_LIBRARY_DIR@:$@LIBRARY_PATH_VAR@
+export @LIBRARY_PATH_VAR@="$this_dir/@KWIVER_DEFAULT_LIBRARY_DIR@:@LIBRARY_PATH_VAR@"
```

Comment:
This change is independent from the rest of this branch.  Is this a bug fix?

@linus-sherrill does removing the `$` make sense here?  I think `LIBRARY_PATH_VAR` expands to a variable name right?

Reply:
Good catch. That seems wrong. I'll add the $ back.

Reply:
I was getting an error from shellcheck on $@ and did not pay enough attention in "fixing it".